.github/workflows/ansible-lint.yml

```diff
@@ -1,11 +1,12 @@
 ---
-# Copyright (C) 2022 Maciej Delmanowski <drybjed@gmail.com>
-# Copyright (C) 2022 DebOps <https://debops.org/>
+# Copyright (C) 2022-2025 Maciej Delmanowski <drybjed@gmail.com>
+# Copyright (C) 2022-2025 DebOps <https://debops.org/>
```

Comment:
@drybjed Nice to see you around!

Hint to simplify maintenance: I now only track the year of initial publication.

https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code#why-not-bump-the-year-on-change explains this best.

Example statement for contributing guide: https://github.com/opening-hours/opening_hours.js/tree/develop?tab=readme-ov-file#license